### DISCUSSION TITLE
How to add a cache policy

### DISCUSSION THREAD

**Discussion by timb-103** [nuxt/nuxt#16289] [Questions] [answered]
### Environment

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v16.14.0`
- Nuxt Version:     `3.0.0`
- Nitro Version:    `1.0.0`
- Package Manager:  `npm@8.5.4`
- Builder:          `vite`
- User Config:      `app`, `css`, `modules`, `googleFonts`, `nitro`
- Runtime Modules:  `@nuxtjs/google-fonts@3.0.0-1`
- Build Modules:    `-`
------------------------------

### Reproduction

```
export default defineNuxtConfig({
  css: ['@/assets/css/global.css'],
  modules: ['@nuxtjs/google-fonts'],

  googleFonts: {
    families: {
      Poppins: [300, 400, 500, 600, 700, 800, 900],
    },
    display: 'swap',
  },

  nitro: {
    compressPublicAssets: true,
  },
})
```

### Describe the bug

Pagespeed insights is telling me to serve static assets with an efficient cache policy, but not sure how to do this in nuxt 3 or if it's possible?

I've added the compressPublicAssets flag to nitro and it does generate the .gz files in the output directory - but when visiting the website it's not loading them as seen in the image below:

![image](https://user-images.githubusercontent.com/105137815/203914308-b9cd6679-26f7-4fd7-b0a0-ab0fdbeff673.png)

**Comment by bot08** [comment, marked as answer]
```
export default defineNuxtConfig({
  routeRules: {
    '/_nuxt/**': { headers: { 'cache-control': 's-maxage=86400' } },
  }
})
```
Something like this?

**Reply by timb-103** [reply]
Perfect! Just had to change it to 'max-age=86400'

Thanks :)
